BLSQ/iaso
# IA-2738: Add "Teams" filter on Completeness stats page

Explain what problem this PR is resolving
- Add a teams filter on completeness stats
Related JIRA tickets : [IA-2738](https://bluesquare.atlassian.net/browse/IA-2738)
## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Add a multiselect component on completeness stats at frontend
- Apply teams filter on backend(instance_created_by__teams__in)

## How to test
- Create two submissions with different users(created_by)
- Link each user to a different team 
- Filter completess stats on team and check the total_instances value

## Print screen / video

Upload here print screens or videos showing the changes
[Screencast from 2024-03-18 17-48-03.webm](https://github.com/BLSQ/iaso/assets/19631540/af42b5d3-d91d-47db-881d-17e09e7dc8bd)


[IA-2738]: https://bluesquare.atlassian.net/browse/IA-2738?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
